services/migration_service/migration_files/20210202145952_add_runs_idx_ts_epoch_flow_id.sql

```diff
@@ -0,0 +1,60 @@
+-- +goose Up
+-- +goose StatementBegin
+SELECT 'up SQL query';
+
+-- Others
+
+CREATE INDEX runs_v3_idx_ts_epoch ON runs_v3 (ts_epoch);
```

Comment:
note that creating an index using the goose script will lock the table and cause some amount of downtime. The runtable is relatively small so not really a major issue however we are working on an alternative way to add indices concurrently